device-types/Cisco/DS-C9148S-D48PSK9.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148S 16G 48 FC
+slug: cisco-ds-c9148s-d48psk9
+part_number: DS-C9148S-D48PSK9
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 17.42
+weight_unit: lb
+comments: '[Cisco MDS 9148S 16G Multilayer Fabric Switch Data Sheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148s-16g-multilayer-fabric-switch/datasheet-c78-731523.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+module-bays:
+  - name: Slot 1
+    label: DS-C9148S-K9-SUP
+    position: '1'
+    desciption: '2/4/8/16 Gbps FC/Supervisor'
+  - name: Slot 2
+    position: '2'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 3
+    position: '3'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 4
+    position: '4'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 5
+    position: '5'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 6
+    position: '6'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 7
+    position: '7'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
```

Comment:
Is this what shows in `show inventory`?

I know on our Nexus 93180YC-FX3 it is not...

```
# show inventory
NAME: "Chassis",  DESCR: "Nexus9000 C93180YC-FX3 Chassis"
PID: N9K-C93180YC-FX3    ,  VID: V01 ,  SN: ###########

NAME: "Slot 1",  DESCR: "48x10/25G + 6x40/100G Ethernet Module"
PID: N9K-C93180YC-FX3    ,  VID: V01 ,  SN: ###########

NAME: "Power Supply 1",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Power Supply"
PID: NXA-PAC-650W-PI     ,  VID: V04 ,  SN: ###########

NAME: "Power Supply 2",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Power Supply"
PID: NXA-PAC-650W-PI     ,  VID: V04 ,  SN: ###########

NAME: "Fan 1",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 2",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 3",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 4",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A
```

Reply:
# show inventory
NAME: "Chassis",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis " 
PID: DS-C9148S-K9        ,  VID: V02 ,  SN: XXXXXXXXXXX          

NAME: "Slot 1",  DESCR: "2/4/8/16 Gbps FC/Supervisor"           
PID: DS-C9148S-K9-SUP    ,  VID: V02 ,  SN: XXXXXXXXXXX          

NAME: "Slot 2",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply"
PID: DS-C48S-300AC       ,  VID: V00 ,  SN: XXXXXXXXXXX          

NAME: "Slot 3",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply"
PID: DS-C48S-300AC       ,  VID: V00 ,  SN: XXXXXXXXXXX          

NAME: "Slot 4",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module"
PID: DS-C48S-FAN         ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 5",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module"
PID: DS-C48S-FAN         ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 6",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module"
PID: DS-C48S-FAN         ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 7",  DESCR: "MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module"
PID: DS-C48S-FAN         ,  VID: V00 ,  SN: N/A